shared_python/TagValidator.py

```diff
@@ -117,10 +121,20 @@ def correct_tag_type(self, tag_type):
         return None
 
     def correct_tag(self, tag, tag_type):
+        if (not tag):
+            return None
+
         # Attempt self correction by uppercasing every first character
-        if (self.classify_tag(tag.title(), tag_type) > 0):
-            self.print_tag_correction(tag, tag.title())
-            return tag.title()
+        tag_correction = tag.title()
+        if (self.classify_tag(tag_correction, tag_type) > 0):
+            self.print_tag_correction(tag, tag_correction)
+            return tag_correction
+
+        # Attempt self correctioni by replacing ampersands with 'and'
```

Comment:
Extra `i` in there gives it an Italian feel 😄

Reply:
Lol. I'll fix that tonight.